tests/test_extensions/test_sklearn_extension/test_sklearn_extension.py

```diff
@@ -2014,6 +2014,10 @@ def test_sklearn_serialization_with_none_step(self):
         with self.assertRaisesRegex(ValueError, msg):
             self.extension.model_to_flow(clf)
 
+    @unittest.skipIf(
+        LooseVersion(sklearn.__version__) < "0.20",
+        reason="columntransformer introduction in 0.20.0",
```

Comment:
There's no column transformer happening here, right?

Reply:
We do have it [here](https://github.com/openml/openml-python/blob/f0ff562d9ca428fc6bae0c52ee6694e2f0a9fdec/tests/test_extensions/test_sklearn_extension/test_sklearn_extension.py#L2049), right?

Reply:
Ha, I didn't see that the unit test continues, sorry